### PR TITLE
benchmarks/transfer: configurable funding address

### PR DESCRIPTION
**Pull request by ptrus**

### CODE REVIEW

File: benchmark/benchmarks/transfer/transfer.go [hunk 1]
```diff
@@ -398,6 +402,8 @@ func Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&flagFaucetURL, cfgFaucetURL, "", "Faucet private endpoint URL")
 	cmd.Flags().BoolVar(&flagWatchNewHead, cfgWatchNewHead, false, "Subscribe for `newHeads` events")
 	cmd.Flags().BoolVar(&flagQueryNonces, cfgQueryNonces, false, "Query explicitly for account nonces")
+	cmd.Flags().StringVar(&flagFundingAccount, cfgFundingAccount, "533d62aea9bbcb821dfdda14966bb01bfbbb53b7e9f5f0d69b8326e052e3450c", "Hex encoded funding account key")
+	cmd.Flags().StringVar(&flagFundingAccountAddress, cfgFundingAccountAddress, "0x7110316b618d20d0c44728ac2a3d683536ea682b", "Funding account address")
 
 	for _, v := range []string{
 		cfgFaucetURL,
```
Comment (nhynes) [inline benchmark/benchmarks/transfer/transfer.go:409]: you sure you don't need to add `cfgFundingAccount` and `cfgFundingAccountAddress`?

File: benchmark/benchmarks/transfer/transfer.go [hunk 2]
```diff
@@ -407,21 +413,14 @@ func Init(cmd *cobra.Command) {
...
+	privKey, err := crypto.HexToECDSA(flagFundingAccount)
 	if err != nil {
-		panic(err)
+		panic(fmt.Sprintf("error parsing funding account private key: %v", err))
 	}
 
 	// Sanity check the addresss.
 	addr := privKeyToAddress(privKey)
-	if strings.ToLower(addr.Hex()) != fundingAddress {
+	if strings.ToLower(addr.Hex()) != flagFundingAccountAddress {
```
Comment (nhynes) [inline benchmark/benchmarks/transfer/transfer.go:423]:
```suggestion
	if strings.ToLower(addr.Hex()) != strings.ToLower(flagFundingAccountAddress) {
```
there's no guarantee that the user won't pass in a lower-cased address

**Review by nhynes — CHANGES_REQUESTED** [review]

File: benchmark/benchmarks/transfer/transfer.go (hunk 1 above)
Comment (ptrus) [reply benchmark/benchmarks/transfer/transfer.go:409]: thanks for noticing :-)

File: benchmark/benchmarks/transfer/transfer.go [hunk 3]
```diff
@@ -398,30 +402,27 @@ func Init(cmd *cobra.Command) {
...
 	cmd.Flags().BoolVar(&flagQueryNonces, cfgQueryNonces, false, "Query explicitly for account nonces")
+	cmd.Flags().StringVar(&flagFundingAccount, cfgFundingAccount, "533d62aea9bbcb821dfdda14966bb01bfbbb53b7e9f5f0d69b8326e052e3450c", "Hex encoded funding account key")
+	cmd.Flags().StringVar(&flagFundingAccountAddress, cfgFundingAccountAddress, "0x7110316b618d20d0c44728ac2a3d683536ea682b", "Funding account address")
 
 	for _, v := range []string{
 		cfgFaucetURL,
 		cfgWatchNewHead,
 		cfgQueryNonces,
+		flagFundingAccount,
+		flagFundingAccountAddress,
```
Comment (nhynes) [inline benchmark/benchmarks/transfer/transfer.go:413]: you sure this shouldn't be `cfg`?

Comment (ptrus) [reply benchmark/benchmarks/transfer/transfer.go]: that's awk. yeah of course!

**Review by nhynes — APPROVED** [review]

**Pull request merged**